iot/manager/manager.js

```diff
@@ -577,24 +594,30 @@ const clearRegistry = async (client, registryId, projectId, cloudRegion) => {
 const deleteRegistry = async (client, registryId, projectId, cloudRegion) => {
   // [START iot_delete_registry]
   // Client retrieved in callback
-  // getClient(serviceAccountJson, function(client) {...});
   // const cloudRegion = 'us-central1';
   // const projectId = 'adjective-noun-123';
   // const registryId = 'my-registry';
-  const parentName = `projects/${projectId}/locations/${cloudRegion}`;
-  const registryName = `${parentName}/registries/${registryId}`;
-  const request = {
-    name: registryName,
-  };
 
-  try {
-    const res = await client.projects.locations.registries.delete(request);
+  const iot = require('@google-cloud/iot');
+
+  const iotClient = new iot.v1.DeviceManagerClient({
+    // optional auth parameters.
+  });
 
+  const registryName = iotClient.registryPath(
+    projectId,
+    cloudRegion,
+    registryId
+  );
+  try {
+    const responses = await iotClient.deleteDeviceRegistry({
+      name: registryName,
+    });
+    console.log(responses);
     console.log('Successfully deleted registry');
-    console.log(res);
   } catch (err) {
     console.log('Could not delete registry');
```

Comment:
Collapse error into one line:

`console.log('Could not delete registry: ', err);`